### PR TITLE
Attend buttons improvement

### PR DESCRIPTION
**Pull request by xundeenergie**
To make attend-buttons for events work the same way as the like/dislike/share-buttons work now, this pull-request is needed. It completes the new functionality with discoupled buttons from full-page-rendering a bit more.

### CODE REVIEW

File: src/App/Page.php [hunk 1]
```diff
@@ -256,6 +256,7 @@ private function initHead(App $app, Arguments $args, L10n $l10n, IManageConfigVa
 			'$likeError'       => $l10n->t('Like not successfull'),
 			'$dislikeError'    => $l10n->t('Dislike not successfull'),
 			'$announceError'   => $l10n->t('Sharing not successfull'),
+			'$attendError'     => $l10n->t('Attendation not successfull'),
```
Comment (MrPetovan) [inline src/App/Page.php]:
```suggestion
			'$attendError'     => $l10n->t('Attendance unsuccessful'),
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Please regenerate the messages.po file.

**Comment by xundeenergie** [comment]
> Please regenerate the messages.po file.

Ok.
Must do it tomorrow... comitted via webapp...

File: view/theme/frio/js/theme.js [hunk 2]
```diff
@@ -802,15 +811,21 @@ function doActivityItemAction(ident, verb, un) {
 	.success(function(data){
 		$('img[id^=waitfor-' + verb + '-' + ident.toString() + ']').remove();
 		if (data.status == 'ok') {
+			if (verb.indexOf('attend') === 0) {
+				$('button[id^=attend][id$=' + ident.toString() + ']').removeClass('active')
+				$('button#attendyes-' + ident.toString()).attr('onclick', 'javascript:doActivityItemAction(' + ident +', "attendyes" )');
+				$('button#attendno-' + ident.toString()).attr('onclick', 'javascript:doActivityItemAction(' + ident +', "attendno" )');
+				$('button#attendmaybe-' + ident.toString()).attr('onclick', 'javascript:doActivityItemAction(' + ident +', "attendmaybe" )');
```
Comment (annando) [inline view/theme/frio/js/theme.js]: Can you fix the superfluous spaces in front of the brackets?

**Comment by xundeenergie** [comment]
> Please regenerate the messages.po file.

done

**Pull request merged**